api/src/test/java/org/openmrs/validator/PersonValidatorTest.java

```diff
@@ -231,5 +234,25 @@ public void validate_shouldNotFailWhenDeathCauseNotCodedIsSet() {
 
 		Assert.assertFalse(errors.hasErrors());
 	}
+	
+	@Test
+	public void validate_shouldNotFailWhenOneAddressIsNull() {
```

Comment:
@dkayiwa should that test be removed as well?  Especially in context of this content I think it should be part of this PR: https://issues.openmrs.org/browse/TRUNK-5473?focusedCommentId=256840&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-256840

Reply:
Tests to catch possible refactoring bugs, can stay. 😊